Validation/RecoTrack/python/HLTmultiTrackValidator_cfi.py

```diff
@@ -9,16 +9,7 @@
 hltMultiTrackValidator.label   = ['hltPixelTracks']
 hltMultiTrackValidator.beamSpot = 'hltOnlineBeamSpot'
 hltMultiTrackValidator.ptMinTP  =  0.4
-hltMultiTrackValidator.lipTP    = 30.0
-hltMultiTrackValidator.tipTP    = 60.0
-hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.ptMin =  0.9
-hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.lip   = 30.0
-hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.tip   = 60.0
-hltMultiTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsEta  = hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.clone()
-hltMultiTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPhi  = hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.clone()
-hltMultiTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPt   = hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.clone(ptMin=0.4)
-hltMultiTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXR = hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.clone()
-hltMultiTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXZ = hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.clone()
+hltMultiTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPt.ptMin = 0.4
```

Comment:
sorry I have a very basic question. Isn't this enlarging the phase-space on which to compute efficiency also for the pixelTracks (that use the same configuration)? Is this what we really want?

Reply:
If you refer to the cut on pT, this was there already for the plots vs pT.
For all other plots, the cut was and is at 0.9 GeV.
Did you refer to anything else?

Reply:
I am referring to the tip cut.

Reply:
The TIP cut applied according to this PR in phase-1 would be 2.5 cm (as in offline), instead of 30 cm:
https://github.com/cms-sw/cmssw/blob/master/Validation/RecoTrack/python/TrackingParticleSelectionsForEfficiency_cff.py#L31

This would be a reduction in phase-space, which is actually more "meaningful" for pixel(-seeded) tracks, since it restricts the phase-space to tracks we are actually trying to reconstruct. Do you agree?